README.md

```diff
@@ -106,7 +94,8 @@ To get involved in Apache Beam:
 * [Subscribe](mailto:dev-subscribe@beam.apache.org) or [mail](mailto:dev@beam.apache.org) the [dev@beam.apache.org](http://mail-archives.apache.org/mod_mbox/beam-dev/) list.
 * Report issues on [JIRA](https://issues.apache.org/jira/browse/BEAM).
 
-We also have a [contributor's guide](https://beam.apache.org/contribute/contribution-guide/).
+Instructions for building and testing Beam itself
+are in the [contributor's guide](CONTRIBUTING.md).
```

Comment:
This line is going to name CONTRIBUTING.md as “contributor’s guide”, which conflicts with the page in the wiki. How about just naming “CONTRIBUTING.md”?

Reply:
One or the other of these is going away. I'm not sure which yet.

Reply:
Kenn responded to the email thread to have  CONTRIBUTING.md link to https://beam.apache.org/contribute/contribution-guide/